src/main/java/com/adobe/cq/testing/client/security/AbstractProfile.java

```diff
@@ -17,6 +17,7 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import org.apache.sling.testing.clients.ClientException;
+import org.jetbrains.annotations.NotNull;
```

Comment:
No, you probably want this: `javax.validation.constraints.NotNull`

Reply:
javax.validation.constraints.NotNull requires adding another dependency to the project.

Since I see it is not used, I'll drop the annotations.